src/services/gmail/service.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "gmail",
+    "name": "Gmail",
+    "url": "https://accounts.google.com"
```

Comment:
hmm? this URL does go to GMail

Reply:
yes direct to gmail,have tried different browsers

Reply:
ah right if you're not logged in already :+1: